gno.land/cmd/gnoland/testdata/addpkg.txtar

```diff
@@ -0,0 +1,32 @@
+# test for add package
+
+## start a new node
+gnoland start
+
+## add bar.gno package located in $WORK directory as gno.land/r/foobar/bar
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/foobar/bar -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+## execute Render
+gnokey maketx call -pkgpath gno.land/r/foobar/bar -func Render -gas-fee 1000000ugnot -gas-wanted 2000000 -args '' -broadcast -chainid=tendermint_test test1
+
+## NOTE: In some rare cases, the previous command hasn't finished. Wait a bit to
+## ensure that the command has been correctly executed.
+## XXX: Find a more convenient and reliable way to ensure that the previous
+## command has been executed.
+sleep 500ms
```

Comment:
Hmm, no, technically `txtar` _does_ execute this sequentially. I'll update my comment to make it clearer. 
Actually, this isn't consistent; it only appeared [once](https://github.com/gnolang/gno/actions/runs/6382204900/job/17320300885?pr=1117#step:5:130) and that was on the CI. you can find the log `e19dbf79.log` in [logs-test-gnoland-go1.21.x artifact](https://github.com/gnolang/gno/suites/16788722555/artifacts/958824404)) but I wasn't able to find any pertinent  message to guess whats actually happening. There's a brief window where it look like the previous 'addpkg' command hasn't finished executing completely. but due to inconsistency , I'm finding it hard to debug, If you (or anyone else) can assist me with this, I'd greatly appreciate it 🥺. 
(It can also maybe due to `gnoland start` that has not entirely finished, but im really not sure about that...)

note: like in shell commands, testscripts [support](https://github.com/rogpeppe/go-internal/blob/v1.11.0/testscript/cmd.go#L233-L248) `&` at the end for explicit bg task 🙂

Reply:
the sleep was miss placed, it should be between the two commands (i've updated the file). But that doesn't change my previous message, the problem is still there.

Reply:
- gnoland start was the main problem 
- sleep method was replaced by a listening event system waiting for the first block (thanks @thehowl  🙏)!